service/EscolaService.cs

```diff
@@ -50,43 +51,94 @@ public List<int> CadastrarEscolaViaPlanilha(MemoryStream planilha)
 
                     while (!parser.EndOfData)
                     {
-                        string[] linha = parser.ReadFields();
-                        if (!primeiralinha)
+                        try
                         {
-                            primeiralinha = true;
-                            continue;
+                            string[] linha = parser.ReadFields();
```

Comment:
Esse método tem uma certa compelexidade em regra de negócio, será que daria pra colocar mais um comentário explicando o início do método. (Algo parecido como vocês fizeram lá embaixo